client/app/components/compose.coffee

```diff
@@ -311,9 +312,13 @@ module.exports = Compose = React.createClass
 
         # new draft
         else
-            state = messageUtils.makeReplyMessage @props.selectedAccountLogin,
-                @props.inReplyTo, @props.action,
-                @props.settings.get('composeInHTML')
+            state = messageUtils.makeReplyMessage(
+                @props.selectedAccountLogin,
+                @props.inReplyTo,
+                @props.action,
+                @props.settings.get('composeInHTML'),
+                @props.selectedAccount.get('signature')
```

Comment:
We already have the account list (for account selector) and current account id, so I think adding `selectedAccount` to the props is useless. You can use `@props.accounts.get(@props.selectedAccountID).get('signature)'`.